packages/k8s/tests/run-container-step-test.ts

```diff
@@ -23,6 +27,47 @@ describe('Run container step', () => {
     expect(exitCode).toBe(0)
   })
 
+  it('should run pod with extensions applied', async () => {
```

Comment:
leave a `//` explaining that we only assert successful run here not individual overrides